docs/api/methods/js_kanban_settheme_method.md

```diff
@@ -0,0 +1,48 @@
+---
+sidebar_label: setTheme()
+title: setTheme Method
+description: You can learn about the setTheme method in the documentation of the DHTMLX JavaScript Kanban library. Browse developer guides and API reference, try out code examples and live demos, and download a free 30-day evaluation version of DHTMLX Kanban.
+---
+
+# setTheme()
+
+### Description
+
+@short: Applies a new theme to Kanban dynamically (with reinitialization)
+
+### Usage
+
+~~~jsx {}
+setTheme(theme?: string): void;
+~~~
+
+### Parameters
+
+- `theme` - (optional) a theme to be applied to Kanban. There are 3 available themes:
```

Comment:
so this is `name`, which is, yeah, optional and a string
the second property of the object is `font`, optional and boolean. it's set if the user wants to load our fonts too. `true` by default